provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -312,7 +312,9 @@ data:
         "service-vlan": {{ config.net_config.service_vlan|json }},
         "kubeapi-vlan": {{ config.net_config.kubeapi_vlan|json }},
         "pod-subnet": {{ config.net_config.pod_subnet|json }},
+        {% if not config.aci_config.disable_node_subnet_creation %}
         "node-subnet": {{ config.net_config.node_subnet|json }},
+        {% endif %}
```

Comment:
We need to populate this regardless of whether we create the ACI subnet or not.

Reply:
Added back!